website/docs/faqs/Runs/checking-logs.md

```diff
@@ -10,7 +10,7 @@ To check out the SQL that dbt is running, you can look in:
 
 * dbt Cloud:
   * Within the run output, click on a model name, and then select "Details"
-* dbt CLI:
+* dbt Core CLI:
```

Comment:
Being these are directories, we could drop "CLI" technically you can navigate to them in vscode or via finder:
```suggestion
* dbt Core:
```